guides/doc-Provisioning_Guide/topics/Networking.adoc

```diff
@@ -381,6 +381,85 @@ Before you begin, ensure that the following conditions are met:
 * The NIC is capable of PXE booting. For more information, see http://ipxe.org/appnote/hardware_drivers
 * The NIC is compatible with iPXE
 
+==== Example WIP of Lzap's Blogpost
+
+Using {Project} you can set up and use iPXE if you have the following restrictions that prevent you from using PXE:
+
+* On networks with unmanaged DHCP servers,
+* If the PXE service is blacklisted on your network or restricted by a firewall
+* When the TFTP UDP-based protocol is not reliable, for example, on low-bandwidth networks
+
+With {Project}, you can set up PXE over TFTP to download the `ipxe.efi` file. When the file downloads, all communication continues using HTTP. {Project} uses the iPXE provisioning script either to load an operating system installer or the next entry in the boot order.
```

Comment:
"PXE over TFTP" is a weird one. I'd probably say that "you can set up hosts to download iPXE (ipxe.efi or undionly.kxpe)".